opm/material/fluidmatrixinteractions/EclMaterialLawManager.hpp

```diff
@@ -350,6 +351,14 @@ class EclMaterialLawManager
     const EclEpsScalingPointsInfo<Scalar>& oilWaterScaledEpsInfoDrainage(size_t elemIdx) const
     { return oilWaterScaledEpsInfoDrainage_[elemIdx]; }
 
+    template<class Serializer>
+    void serializeOp(Serializer& serializer)
+    {
+        for (auto& mat : materialLawParams_) {
```

Comment:
What about the dirMaterialLawParams_ member?

Reply:
as far as i can tell it only contains static data initialized from the EclipseState.